g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/conf/RequestLogInterceptor.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019-2023 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.l10n.conf;
+
+import com.vmware.vip.common.constants.ConstantsChar;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Arrays;
+import java.util.List;
+
+public class RequestLogInterceptor extends HandlerInterceptorAdapter {
```

Comment:
i think it's not necessary to create a new interceptor which can decrease the performance of request handling, so can you add the log message about request-id in CspAuthInterceptor?

Reply:
only can add the log in CollectSourceValidationInterceptor and have done!